src/lumigo_log_shipper/lumigo_shipper.py

```diff
@@ -14,12 +14,17 @@
 from lumigo_log_shipper.utils.sts import ChinaMissingEnvVar
 
 
-def ship_logs(aws_event: dict, programmatic_error_keyword: str = None) -> int:
+def ship_logs(
+    aws_event: dict,
+    programmatic_error_keyword: str = None,
+    exclude_filters: List[str] = None,
```

Comment:
```suggestion
    exclude_filters: Optional[List[str]] = None,
```
It's weird that mypy didn't catch it, it's like saying `x: int = "hello"` 🤔